.travis.yml

```diff
@@ -3,30 +3,30 @@ services:
     - secure: "Pu8tfrmmWAnQYmgZ9fmiosVj22XeoZ6mSwSKe3knjmLQYC8A/9VJSFjpZn11dqUQokhZMbgbTYthp4c4/JW7cy8nGWykg7a/osiWqNAuPYiJl+oUmPJcxW0Enagq8p5+RNbFfeND4GyX5ZdBi0xRaGh2CgUbmVYxbaSfq0Sbkfzgl4HkF8uqBPkjdfQLBJ/kAe5prXkAAAyu58LYvPNjpMoOSlOS+c7F+XNokMLzlsL+sKE/CC2l6tLjDslxLX5CzW8jA1vBkqhThAO0Fe5UYv+6AqDBfsYBro01bgmfMR9F/IwBkOH3eFcs/icm5BOeYaQ2It5YEhbSW1CY8U/iFmKkpKWe5q8Jw4fgRPB4QTcVu/M0ffvWtlaTwCV0T9kE93imfQidfrRSTbnDnZwunxpn3Ymx8OAhEzeLndq56wq0ShIJTkh65mtkHVf+75dQ6F4nHwB+RkhHT6ggGWGUNcYOkMb+UQq4bzpGmQNJC9zdFtqOp1EcB5WnlOrDxbyxJ8yrYNvKvJ8+wi1K+Z/RZvRCCE+P2AJQc2mMlgKelP7WUPnhcNRTEyMa6Q6L0EklluVRR3b4N7nbpqlwU+lYU2pyJ764ULNXJJfk7CmC3dhI3PBQ/BreFYNNd7TujrNK53T41VAYfHxhIoz8lwsHT/fOIKAKE7LLqXvbR3Tp+wo="
 
 install:
-  - make image
-  - docker images
   - curl -L https://github.com/arminc/clair-scanner/releases/download/v8/clair-scanner_linux_amd64 > clair-scanner
   - chmod +x clair-scanner
   - sudo mv clair-scanner /usr/local/bin
-  
-script:
-  - curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64 && chmod +x container-structure-test-linux-amd64 && sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
-  - container-structure-test test --image pegasystems/pega-ready:qualitytest --config tests/pega-web-ready-testcases.yaml
-  - container-structure-test test --image pegasystems/pega-ready --config tests/pega-web-ready-release-testcases.yaml
   - docker run -d --name db arminc/clair-db
   - docker run -p 6060:6060 --link db:postgres -d --name clair arminc/clair-local-scan:latest
-  - clair-scanner -w tests/cve-scan-whitelist.yaml -c "http://127.0.0.1:6060" --threshold="Medium" --ip "$(ip -4 addr show eth0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')" pegasystems/pega-ready:latest
+  - curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64 && chmod +x container-structure-test-linux-amd64 && sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
+  
+script:
+  - make test
+  - docker images
+  - clair-scanner -w tests/cve-scan-whitelist.yaml -c "http://127.0.0.1:6060" --threshold="Medium" --ip "$(ip -4 addr show eth0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')" $IMAGE_NAME:latest
 
 before_deploy:
   - docker login -u="$DOCKER_USER" -p="$DOCKER_PASS"
+  - docker tag $IMAGE_NAME:latest $IMAGE_NAME:$VERSION #Tag for current version
 
 deploy:
   provider: script
   script: make push
   on:
-    repo: pegasystems/docker-pega-web-ready
-    branch: master
+    repo: pegasystems/docker-pega-web-ready
```

Comment:
With this, are we allowing all successful pull requests to be merged to the actual docker hub repository before merge to master?

Reply:
Based on my reading of the travis docs, it will only execute on merged branches - so no PRs.  I'll update it with a condition for a bit more clarity and to prevent the possibility of a false push.

https://docs.travis-ci.com/user/deployment

Reply:
I updated the deploy condition to build when the branch is "master" or it starts with "v" (ie, v2.0.0).   Unfortunately this is difficult to verify as working in a branch because it is a condition that only executes after the PR is merged.  If this poses a problem post-merge I will fix or revert.